Sources/ShopifyAcceleratedCheckouts/Wallets/ApplePay/ApplePayConfiguration.swift

```diff
@@ -60,24 +60,40 @@ extension ShopifyAcceleratedCheckouts {
         ///         For shops requiring customer accounts, include `.email` in the array.
         public let contactFields: [RequiredContactFields]
 
+        /// Countries supported for Apple Pay shipping addresses.
+        ///
+        /// This property allows merchants to restrict which countries are available for
+        /// shipping addresses during the Apple Pay checkout flow. Uses ISO 3166-1 alpha-2
+        /// country codes (e.g., "US", "CA", "GB").
+        ///
+        /// When nil or empty, all countries are allowed, maintaining backward compatibility.
```

Comment:
I don't think we need to mention backwards compatibility - just that when nil all countries are allowed